### PR TITLE
Update V8 to 7.0.247

### PR DESCRIPTION
**Pull request by ameerthehacker**

### CODE REVIEW

File: depot_tools/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# The `recipes` attribute controls which additional files are included in the
+# recipes bundle when running the `//recipes/recipes.py bundle` command. These
+# files will also be available to other repos' recipes when they pull in
+# depot_tools as a recipe dependency.
```
Comment (ry) [inline depot_tools/.gitattributes]: Don't include things unrelated to V8

**Comment by ry** [comment]
Thanks! But please only include changes related to V8.

File: depot_tools/.gitattributes (hunk 1 above)
Comment (ameerthehacker) [reply depot_tools/.gitattributes]: Sure @ry I will fix them

**Comment by ameerthehacker** [comment]
Hi @ry I have removed the unwanted stuffs and have added only v8 in this pull request!

**Pull request closed without merging**